presto-postgresql/src/main/java/io/prestosql/plugin/postgresql/PostgreSqlClient.java

```diff
@@ -259,7 +260,7 @@ protected ResultSet getTables(Connection connection, Optional<String> schemaName
         }
     }
 
-    private Map<String, Integer> getArrayColumnDimensions(Connection connection, JdbcTableHandle tableHandle)
+    private static Map<String, Integer> getArrayColumnDimensions(Connection connection, JdbcTableHandle tableHandle)
```

Comment:
in the "Cleanup warnings" commit, could you move `SORTED_MAPPER` and `JSON_FACTORY` fields to the top of the class?